### ISSUE TITLE
SoD extra attacks from wild strikes not reseting the swing timer indicator

### ISSUE THREAD

**Issue by tlecas** [watchyoursixx/WeaponSwingTimer-SixxFix#64]
Currently the addon doesn't reset the melee swing timer when you get a wild strikes proc (SoD druid windfury equivalent).
I'm unable to test if it works properly with other sources of extra attacks (WF, sword specs, HoJ).

**Comment by watchyoursixx** [comment]
Hello I know it's been a while, but if you or anyone else can confirm the spell ID of the proc I can add it

**Comment by tlecas** [comment]
407973 https://www.wowhead.com/classic/spell=407973/wild-strike
But it's not an exception, the addon doesn't handle extra attacks correctly on vanilla versions of the game.
It shouldn't exclude extra attacks from resetting the swing timer, that's TBC and beyond behaviour.
In vanilla extra attacks simply complete your current swing timer, so you don't need extra logic for extra attack procs.

**Comment by watchyoursixx** [comment]
https://github.com/watchyoursixx/WeaponSwingTimer-SixxFix/blob/99de605e4803cb26e0bda43155a86eda0411ff9a/WeaponSwingTimer_Player.lua#L127

This is how extra attacks are currently handled which may or may not be correct. I'll do some more digging into it

**Comment by watchyoursixx** [comment]
Do you have a log file with this? Or able to make a quick one and share it? Just need to see how it triggers in the combat log

**Comment by tlecas** [comment]
> [WeaponSwingTimer-SixxFix/WeaponSwingTimer_Player.lua](https://github.com/watchyoursixx/WeaponSwingTimer-SixxFix/blob/99de605e4803cb26e0bda43155a86eda0411ff9a/WeaponSwingTimer_Player.lua#L127)
> 
> Line 127 in [99de605](/watchyoursixx/WeaponSwingTimer-SixxFix/commit/99de605e4803cb26e0bda43155a86eda0411ff9a)
> 
>  -- added check for extra attacks that would accidently reset the swing timer, reset by a sucessful 
> This is how extra attacks are currently handled which may or may not be correct. I'll do some more digging into it

This is correct behaviour for TBC onward but not for vanilla. Unfortunately I dont have any Era characters to demonstrate it with.
https://github.com/magey/classic-warrior/wiki/Windfury-Totem The introduction of this article covers the behaviour of extra attacks in vanilla.
I think you could change line 128 to something like:
`if (event == "SPELL_EXTRA_ATTACKS" and not (WOW_PROJECT_ID == WOW_PROJECT_CLASSIC)) then`
Not sure if this is proper Lua syntax but you get the idea.
